### ISSUE TITLE
Instructions for finding screen ID

### ISSUE THREAD

**Issue by dhouck** [KyrptonaughtMC/Inventory-Sorter#113]
I want to disable sorting with the Traveler’s Backpack mod’s backpacks, but I can’t find the screen ID, and ctrl+clicking doesn’t seem to work (maybe because the sort button overlaps a slot).  Is there a way to find the screen ID to add to the blacklist?

**Comment by ninjaeon** [comment]
> I want to disable sorting with the Traveler’s Backpack mod’s backpacks, but I can’t find the screen ID, and ctrl+clicking doesn’t seem to work (maybe because the sort button overlaps a slot). Is there a way to find the screen ID to add to the blacklist?

Did you ever find a solution?  I'm also having the same issue, using travelersbackpack-fabric-1.20.1-9.1.12.jar on 1.20.1

![image](https://github.com/kyrptonaught/Inventory-Sorter/assets/10052279/48c810ea-5de8-46e4-930a-887ce6cd40c5)

**Comment by dhouck** [comment]
No, I gave up on Travelerʼs Backpack because of various item dupe and deletion bugs, before figuring this out.  Iʼm now using INMIS, which supports the sort button added here.

But better instructions for how to get it removed from screens where it doesnʼt make sense would still be useful.

**Comment by GamerTable** [comment]
Anyone else get this working?

**Comment by ninjaeon** [comment]
> Anyone else get this working?

I didn't.  I ended up replacing Inventory Sorter with another mod that effectively does the same thing but doesn't overlap with Traveler's Backpack.

**Comment by dhouck** [comment]
I kept Inventory Sorter but changed to a different backpack mod because Travelerʼs Backpack has a lot of item dupe and deletion bugs, and the new one happens to work with Inventory Sorter.

**Comment by meza** [comment]
If anyone still needs info on this, you can CTRL+Left Click on the sort button which will then allow you to blacklist the given inventory. It will show a message in chat with a clickable action.

I will be adding proper documentation around this soon

**Comment by meza** [comment]
Now in the new doc! https://inventorysorter.meza.gg/admin-guide#getting-the-screenid
